app/assets/stylesheets/application.css.scss

```diff
@@ -21,6 +21,7 @@
 *= require welcome
 *= require alerts
 *= require listing
+*= require mailers
```

Comment:
Do we need the mailers.scss file to be included in the compiled css?  I think this file contains styles for emails sent by the system, and shouldn't be included in the application's stylesheet.